x-pack/plugin/sql/src/test/java/org/elasticsearch/xpack/sql/planner/QueryTranslatorTests.java

```diff
@@ -396,6 +390,21 @@ public void testTranslateMathFunction_HavingClause_Painless() {
         assertThat(aggFilter.scriptTemplate().params().toString(), endsWith(", {v=10}]"));
     }
 
+    public void testGroupByAndHavingWithFunctionOnTopOfAggregation() {
+        LogicalPlan p = plan("SELECT keyword, max(int) FROM test GROUP BY 1 HAVING " + "abs(max(int)) > 10");
```

Comment:
`HAVING " + "abs(max(int))` - one String only, no need to split it in half.

Reply:
yep, was some leftover